Dfe.Data.SearchPrototype/Infrastructure/Builders/ISearchOptionsBuilder.cs

```diff
@@ -21,6 +21,18 @@ public interface ISearchOptionsBuilder
         /// </returns>
         ISearchOptionsBuilder WithSize(int? size);
 
+        /// <summary>
+        /// Sets the value used to define how many
+        /// records are skipped in the search response (if any).
+        /// </summary>
+        /// <param name="offset">
+        /// The number of initial search results to skip.
+        /// </param>
```

Comment:
What happens when the `offset` is null? 

It seems to be stated routinely within the use case code, but seems to be missing in the infrastructure code? 

Let's state this explicitly, perhaps? Unless there's a reason not to?